tests/python/unit/requirements.txt

```diff
@@ -0,0 +1,38 @@
+pytest~=6.2
+allure-pytest~=2.8
+numpy>=1.15,<1.16
+scipy>=1.2,<1.3
+scikit-learn>=0.20,<0.21
```

Comment:
Make sure this is the same version as the one used by DSS 9

Reply:
@david-behar